src/content/docs/reference/api/git-credentials.md

```
@@ -0,0 +1,69 @@
+### Headers
+
+| Header          | Required    | Description                                |
+| --------------- | ----------- | ------------------------------------------ |
+| `Authorization` | Yes         | Bearer token containing Buildkite OIDC JWT |
+| `Content-Type`  | Recommended | Should be `text/plain`                     |
+
+### Request body
+
+The request body follows Git's credential helper input format:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)